ui/lib/core/addon/templates/components/replication-action-promote.hbs

```diff
@@ -1,28 +1,57 @@
-{{#if (and (eq replicationMode 'dr') (eq model.replicationAttrs.modeForUrl 'secondary'))}}
-  <div class="box is-marginless is-shadowless">
+<div class="action-block is-rounded">
+  <div class="action-block-info">
+    <h4 class="title is-5 is-marginless">
+      Promote cluster
+    </h4>
     <p>
-      This cluster is currently running as a DR Replication Secondary.
-      Promote the cluster to a primary by entering DR Operation Token.
+      Promote this cluster to a {{replicationDisplayMode}} primary
     </p>
-    <div class="field">
-      <label for="dr_operation_token" class="is-label">
-        DR Operation Token
-      </label>
-      <div class="control">
-        {{input class="input" id="dr_operation_token" name="dr_operation_token" value=dr_operation_token}}
-      </div>
-    </div>
-    <ToggleButton
-      @class="is-block"
-      @toggleAttr="showOptions"
-      @toggleTarget={{this}}
-      @openLabel="Hide options"
-      @closedLabel="Options"
-      />
-    {{#if showOptions}}
-      <div class="box is-marginless">
+  </div>
+
+  <div class="action-block-action">
+    <button
+      class="button is-tertiary"
+      onclick={{action (mut isModalActive) true}}
+      data-test-replication-action-trigger
+    >
+      Update
+    </button>
+  </div>
+</div>
+
+
+<Modal
+  @title="Promote cluster?"
+  @onClose={{action (mut isModalActive) false}}
+  @isActive={{isModalActive}}
+  @type="warning"
+  @showCloseButton={{true}}
+>
+  <section class="modal-card-body">
+    <div class="box is-shadowless is-fullwidth is-sideless">
+
+      {{#if (eq replicationDisplayMode "DR")}}
+      <p class="has-bottom-margin-m">
+        To promote this DR Replication Secondary to a primary, enter the DR Operation token.
+      </p>
+      {{/if}}
+      <p class="has-bottom-margin-m">
+        Vault Replication is not designed for active-active usage. Enabling two primaries should never be done, as it can lead to data loss if they or their secondaries are ever reconnected. If the cluster has a primary, be sure to demote it before promoting a secondary.
+      </p>
+
+      <div data-test-promote-dr-inputs>
+        {{#if (eq replicationDisplayMode "DR")}}
```

Comment:
That's a good question, and actually I should be checking `replicationMode` instead since that is not for display purposes. Good catch! I'll post a question about which one we want to standardize on.